VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_breakpoints.scss

```diff
@@ -0,0 +1,18 @@
+@import 'functions';
+
+$breakpoints: (
+  smallest: 0px,
+  extra-small : 481px,
+  small: 641px,
+  medium : 769px,
+  large: 1025px,
```

Comment:
I increased these by 1px as that is more accurate to what was discussed with Neil previously. These are the lower bounds rather than the upper bounds.